.env.example

```diff
@@ -16,24 +16,22 @@ DB_PASSWORD=postgres
 # TELESCOPE_ENABLED=true
 # DEBUGBAR_ENABLED=true
 
-# BROADCAST_DRIVER=log
-# CACHE_DRIVER=file
-# QUEUE_CONNECTION=sync
-# SESSION_DRIVER=database
-# SESSION_LIFETIME=120
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+QUEUE_CONNECTION=sync
+SESSION_DRIVER=database
+SESSION_LIFETIME=120
 
-# REDIS_HOST=127.0.0.1
-# REDIS_PASSWORD=null
-# REDIS_PORT=6379
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
```

Comment:
Why are you uncommenting these? We don't have redis in prod, and the others are default values?

Reply:
doesn't really matter since these are the default values from their respective config files. its custom to keep these values accessible in the .env.example so that new users can setup and have a good idea about what options are available in laravel, where the session is stored (typically file in local but db or redis in prod). I do agree that the redis block might be more confusing than the other one, let me remove it.

Reply:
Correct, but the custom is also to keep them uncommented if they just have the defaults. Uncommenting them, imo, implies we use them (which we don't)

Reply:
these we do use, albeit implicitly:
```
BROADCAST_DRIVER=log
CACHE_DRIVER=file
QUEUE_CONNECTION=sync
SESSION_DRIVER=database
SESSION_LIFETIME=120
```
having them available for the maintainer to manipulate is pretty useful.